docs/sdk/guides/03-execute-remote-smart-contract-call.md

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 3
+---
+
+# Execute Remote Smart Contract Call
+
+This Dapp allows you to call Pangolin Smart Chain's `add(2)` function from Pangoro.
```

Comment:
Pangoro Or Pangoro Smart Chain. They are two different stuff in the Darwinia system. Please check the place they used in the docs.